### DISCUSSION TITLE
Cannot re-enable classic explorer ribbon

### DISCUSSION THREAD

**Discussion by FirmDaKitsune** [Open-Shell/Open-Shell-Menu#2032] [Q&A]
I was messing around with some settings and unchecked the "show classic toolbar" in windows explorer and now I have no way to show it again AND I found NOTHING about how to get it back (I even reinstalled open-shell).

**Comment by PhilipBeinggs** [comment]
Just wanted to second this: I thought I was mad or stupid until I found this post, but I can't see any way to re-enable it either after un-checking "Show classic toolbar" when right-clicking in space to the left of the classic toolbar in Windows Explorer.

I've looked for a Show/Hide option in Classic Explorer Settings -> Toolbar Settings, but I can't see it there or anywhere else.

Are there just two of us who are mad or stupid, among all the Open-Shell users? 😄

**Reply by rixtech** [reply]
Seems there are four of us now 😉

**Comment by PhilipBeinggs** [comment]
OK @FirmDaKitsune, I looked in the registry using RegEdit and found the setting named `ShowedToolbar` in `Computer\HKEY_CURRENT_USER\SOFTWARE\OpenShell\ClassicExplorer`, which is set to `1`. I changed it to `0` and re-opened Windows Explorer and the Classic Explorer Bar reappeared in Windows Explorer:

![image](https://github.com/user-attachments/assets/cfebcecc-6d89-4d13-8da5-ae6a882fc48f)

If anyone else knows of an 'official' way to re-enable it after turning it off using the right-click menu in Windows Explorer, please reply here.

**Reply by githubzoidal** [reply]
I have the same problem. I just did your suggested regedit. But then, how to Quit Explorer? Restart pc? I did do a reinstall just now of Open Shell and toggled what I thought was relevant to the file ribbon but it's still not back. Btw, the Help file says:

_2.Turn on the menu in Explorer – Go to Tools (Alt+T), Folder Options, the View tab, and make sure “Always show menus” is checked.
3.Right click on the menu bar and select “Classic Explorer Bar” to show the toolbar._

BUT I don't see a Tools menu, even with Alt+T, nor does the #3 line work.

**Comment by rixtech** [comment]
@githubzoidal the regedit didn't work for me either.

For the "Quit Explorer", I generally just close the File Explorer window(s), then open a new window to see the Classic Explorer changes. If things still don't look right, I restart the Windows Explorer process from the Task Manager.

The help file worked fine for Windows 8 and 10. But Windows 11 changes everything, right 😄 

**My workaround**

1. Install [Windhawk](https://windhawk.net/) 
2. Install Windhawk [Classic Explorer navigation bar](https://windhawk.net/mods/explorer-frame-classic) mod.
3. Set the Explorer Style to "Classic ribbon UI (no tabs)"
4. Restart Windows Explorer

File _Explorer View -> Options_ now shows the _Classic Explorer Bar_
 
![image](https://github.com/user-attachments/assets/93aa8378-d6fa-4033-ad40-fae045292f77)


The _Classic Explorer Bar_ is back albeit it is now right justified, I guess I can get used to that. 
I fold up the ribbon for a cleaner view.

![image](https://github.com/user-attachments/assets/87f2a94f-f98e-405b-bf52-acdaa0244a03)


Disabling the Windhawk mod returns to the "Moments" update with tabs.
The "New" command bar comes back, which I find jarring. So I'm sticking with the "Classic ribbon UI (no tabs)"

![image](https://github.com/user-attachments/assets/d73e55ea-c414-417c-9d88-e50caa6c84be)

**Reply by rixtech** [reply]
Of course the above broke after the Feb 2026 and April 2026 Windows Updates.

Link to newer solution https://github.com/Open-Shell/Open-Shell-Menu/issues/2365#issuecomment-4411448626

**Comment by dbernardid-ctrl** [comment]
Je suis sous windows 11 25H2 et je n'ai plus ma toolbar depuis un mois et j'essaye par tous les moyens de la retrouver, si quelqu'un a une astuce, je suis preneur.

**Comment by Guenther49** [comment]
Hello, I have also been missing the ClassicExplorer toolbar buttons for some time (Windows 11 24H2).
I tried changing the registry entry described
![001](https://github.com/user-attachments/assets/70f7cbde-9d9a-43b0-a0e6-2c0289dd26e7)
from 1 to 0, but it keeps jumping back to 1.

Then I set the Windhawk Classic Explorer navigation bar mod. with Explorer Style to “Classic ribbon UI (no tabs)”.
Neither with the installation of ClassicShellSetup_4_3_1-de nor with Open_Shell_Setup_v4.4.196 is the navigation bar offered in the Explorer options:
![002](https://github.com/user-attachments/assets/d5fbebab-ebdf-4d57-a36d-29c18ca80f33)
best regards
Günther

**Reply by dbernardid-ctrl** [reply]
il faut espérer qu'une mise a jour soit faite a ce niveau. Es que l'auteur du programme connait ce problème et si il travail pour refaire cette barre, es qu'un email existe pour le contacter. ?

**Reply by kernelpanic31** [reply]
This exact same thing happened to me, it seems to have happened after a minor Windows update, it may be fixed after another Windows Update, it has happened that something gets broken and gets backs to normal after a couple of updates

**Comment by Guenther49** [comment]
Since it stopped working, we've had at least two Windows updates.
I think that, if anything, it can only be solved by the developers.

**Comment by dbernardid-ctrl** [comment]
Bonjour,
J'ai mis à jour un Pc dont la dernière datait du mois de octobre 2025 ou 
était installé la toolBar, et j'ai vu que la mise à jour qui supprimait 
cette ToolBar était la KB50741005 Build 26100.7705 si sa peut vous aider 
?
Cordialement.


------ Message d'origine ------
De "Guenther49" ***@***.***>
À "Open-Shell/Open-Shell-Menu" ***@***.***>
Cc "dbernardid-ctrl" ***@***.***>; "Comment" 
***@***.***>
Date 31/01/2026 18:19:14
Objet Re: [Open-Shell/Open-Shell-Menu] Cannot re-enable classic explorer 
ribbon (Discussion #2032)

**Reply by kernelpanic31** [reply]
Hello.
Yes indeed, i did a little bit of research myself too and found out January 2026 (KB5074105) update was the last one installed on my system, reverted, and it did the work, classic explorer toolbar is back.
Windows has been breaking core UI components lately, so i wouldn't be surprised if they fix it out of nowhere in the coming months, there was a time they broke the browse window when you pressed any "Backup" option for load or export XML configuration files, and they repaired it after a few minor updates.

**Comment by dbernardid-ctrl** [comment]
D'après ce que vous dites votre barre est de retour ? Moi elle ne l'est 
pas, pouvez-vous me dire ce que vous avez fait exactement, a moins que 
j'ai mal compris ?
-----------------------------------


------ Message d'origine ------
De "KernelPanic" ***@***.***>
À "Open-Shell/Open-Shell-Menu" ***@***.***>
Cc "dbernardid-ctrl" ***@***.***>; "Comment" 
***@***.***>
Date 06/02/2026 09:18:45
Objet Re: [Open-Shell/Open-Shell-Menu] Cannot re-enable classic explorer 
ribbon (Discussion #2032)

**Reply by kernelpanic31** [reply]
I simply went to settings -> Update History -> Uninstall updates -> and i looked up for the one i mentioned (KB5074105) clicked uninstall, i let the system clean everything, rebooted one more time after everything was set. I normally have "Windows 7 command bar" Explorer with ExplorerPatcher software, but i had to change it to WIndows 10 Ribbon, and reenable it from the View tab -> Options. I'm using a Windows 11 LTSC edition, so maybe i receive less updates, i don't know, you may have another one that it's causing the problem, i don't usually recommend it but i used Gemini Google AI to look for the Windows updates listed in my "Uninstall Updates"section on the Settings app and tell me which one where installed on January, only KB5074105 was a major release, the other ones were just security updates, those are fine.

**Comment by dbernardid-ctrl** [comment]
En janvier c'est la KB5074103 Build 26100.7623.
Il y a peut-être une alternative concernant cette ToolBar, en faite 
c'est l'explorateur Windows qui nous pose des problèmes avec ces menus 
vraiment dérisoire, c'est Directory Opus il peut créer des barres 
d'outils, mais il faut savoir le paramétrer et c'est pas dans mes 
compétences.
--------------------------------------


------ Message d'origine ------
De "KernelPanic" ***@***.***>
À "Open-Shell/Open-Shell-Menu" ***@***.***>
Cc "dbernardid-ctrl" ***@***.***>; "Comment" 
***@***.***>
Date 06/02/2026 09:38:51
Objet Re: [Open-Shell/Open-Shell-Menu] Cannot re-enable classic explorer 
ribbon (Discussion #2032)

**Comment by dbernardid-ctrl** [comment]
Classic bar explorer.

Bonjour,
J'ai réalisé le fichier .reg comme décrit mais sa ne marche pas, 
peut-être que j'oublie quelque chose ?
Si vous pouvez aidez moi.

Cordialement.
-------------------------------------------------------------
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FeatureManagement\Overrides\8\815149711]
"EnabledState"=dword:00000001
"EnabledStateOptions"=dword:00000000
"note"="Classic explorer bar fix - April 2026 win update"
"Variant"=dword:00000000
"VariantPayload"=dword:00000000
"VariantPayloadKind"=dword:00000000
------------------------------------------------------------


------ Message d'origine ------
De "rixtech" ***@***.***>
À "Open-Shell/Open-Shell-Menu" ***@***.***>
Cc "dbernardid-ctrl" ***@***.***>; "Comment" 
***@***.***>
Date 09/05/2026 10:08:26
Objet Re: [Open-Shell/Open-Shell-Menu] Cannot re-enable classic explorer 
ribbon (Discussion #2032)
